ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -34,9 +34,10 @@ message Commands {
   // Required
   string application_id = 3;
 
-  // Uniquely identified the command. This identifier should be unique for each new command within an
-  // application domain, i.e., the triple (application_id, party, command_id) must be unique.
-  // It can be used for matching the requests with their respective completions.
+  // Uniquely identifies the command.
+  // The triple (application_id, party + act_as, command_id) constitutes the change ID for the intended ledger change,
+  // where party + act_as is interpreted as a set of party names.
+  // The change ID can be used for matching the intended ledger changes with their respective completions.
```

Comment:
(Not introduced by this PR but) I don't think it is meaningful to match completions with ledger changes (rather with submissions).

Reply:
Indeed, we only have a one-to-one mapping between submissions and completions and not between change IDs and completions. However, having both of these available enables the application to decide how it wants to track changes. Let's continue this discussion (if you want) on a separate PR addressing proto documentation.